src/views/AdminSettings.vue

```diff
@@ -628,6 +642,106 @@ export default {
 					return err
 				})
 		},
+
+		saveAll() {
+			this.saving = true
+			this.objectTypes.forEach((objectType) => {
+				this[objectType] = {
+					...this[objectType],
+					loading: true,
+				}
+			})
+
+			// eslint-disable-next-line no-console
+			console.log('Saving all config')
```

Comment:
We kunnen beter `console.info()` gebruiken, dit geeft geen ESLint error en werkt hetzelfde als `console.log()`.

Het klopt ook met wat we willen laten zien, dit is meer info in plaats van logging.